Assets/HoloToolkit/ARCA/Scripts/Utils/Singleton.cs

```diff
@@ -3,7 +3,7 @@
 
 using UnityEngine;
 
-namespace ARCA
+namespace HoloToolkit.ARCapture
 {
     public class Singleton<T> : MonoBehaviour where T : MonoBehaviour
```

Comment:
Is there a reason to create yet another Singleton? Has the MRTK Singleton behaviour deviated so far that it would make porting to it problematic? Perhaps this should be renamed to avoid confusion.

Reply:
Yes your right this was used before ARCA was integrated into the toolkit.  I'll remove this in favour of the HoloToolkit.Unity.Singleton.

Reply:
I think they should use the Singleton provided.